src/js/api/size.js

```diff
@@ -164,3 +164,101 @@ ripe.Ripe.prototype.nativeToSizeB = function(scales, values, genders, options, c
 
     return this._cacheURL(options.url, options, callback);
 };
+
+/**
+ * Converts a size value in the specified scale to the corresponding localized size.
+ * The available scales, genders and sizes can be obtained with the method getSizes.
+ *
+ * @param {String} scale The scale which one wants to convert from.
+ * @param {Number} value The value which one wants to convert.
+ * @param {String} gender The gender of the scale and value to be converted.
+ * @param {Object} options An object of options to configure the request.
+ * @param {Function} callback Function with the result of the request.
+ * @returns {XMLHttpRequest} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype.sizeToLocale = function(scale, value, gender, options, callback) {
+    return this.sizeToLocaleB(scale, value, gender, options, callback);

```

Comment:
This is not correct b is for bulk

Reply:
True but it can actually handle both. Is that a problem? Anyway, will change to `[value]`, etc.